client-src/js-src/cs-client.js

```diff
@@ -103,24 +103,24 @@ class ChromeStatusClient {
     return JSON.parse(rawResponseText.substr(XSSIPrefix.length));
   }
 
-  doGet(resource, body) {
+  async doGet(resource, body) {
```

Comment:
Is there a reason why `async` is needed or is this just to clarify that each of these functions returns a promise?

Reply:
This is needed to avoid *some* of the errors after webtest runs.  More remain, but I am not sure how to fix them.